cmd/notation/init.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+import (
+	_ "github.com/notaryproject/notation-core-go/signature/cose"
+	_ "github.com/notaryproject/notation-core-go/signature/jws"
+)
```

Comment:
We can remove this file since those packages are imported by `internal/envelope/envelope.go` and many other places.

Reply:
removed